nucliadb/src/nucliadb/search/search/rank_fusion.py

```diff
@@ -107,6 +107,16 @@ def _fuse(
         return merged
 
 
+class CombSimple(RankFusionAlgorithm):
+    @rank_fusion_observer.wrap({"type": "legacy"})
+    def _fuse(
+        self, keyword: Iterable[TextBlockMatch], semantic: Iterable[TextBlockMatch]
+    ) -> list[TextBlockMatch]:
+        merged = [*keyword, *semantic]
+        merged.sort(key=lambda r: r.score, reverse=True)
+        return merged
```

Comment:
is this used anywhere? if not, let's remove it no?

Reply:
Ups, leftover! I'll remove it